src/kaleidoscope/driver/Bootloader.h

```diff
@@ -0,0 +1,99 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::driver::bootloader -- Various bootloader drivers
+ * Copyright (C) 2019  Keyboard.io, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <avr/wdt.h>
+
+namespace kaleidoscope {
+namespace driver {
+namespace bootloader {
+
+class Caterina {
+ public:
+  Caterina() {}
+
+  static inline void resetDevice() {
```

Comment:
I commented earlier about the use of `inline` here (and elsewhere) inside class definitions, but I was perhaps unclear. Perhaps I have misunderstood things that I've read, but my understanding was that all functions defined inside the class definition are implicitly `inline`, so the keyword has no effect here. Is there a specific purpose for using it here?

Reply:
Nope, no specific purpose. It's a leftover from an earlier attempt, and I'll be removing them. Thanks!